src/redux/uniqueTokens.js

```diff
@@ -156,25 +182,57 @@ export default (state = INITIAL_UNIQUE_TOKENS_STATE, action) => {
         loadingUniqueTokens: false,
       };
     case UNIQUE_TOKENS_GET_UNIQUE_TOKENS_REQUEST:
-      return {
-        ...state,
-        fetchingUniqueTokens: true,
-      };
+      if (action.showcase) {
+        return {
+          ...state,
+          fetchingUniqueTokens: true,
```

Comment:
just want to double check this flag isn't flipped (like should `fetchingUniqueTokensShowcase` be here, and should `fetchingUniqueTokens` be on line 193?)